geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionStats.java

```diff
@@ -1619,6 +1610,10 @@ public void endReplyWait(long startNanos, long initTime) {
     Breadcrumbs.setProblem(null); // clear out reply-wait errors
   }
 
+  void recordMaxReplyWaitTime(long waitTime) {
+    maxReplyWaitTime.recordMax(waitTime);
```

Comment:
Since you can now unit test the code in its own class, consider getting rid of this one liner

Reply:
yup, that's the plan.  wanted to get your buy in on the class before i did too much work.